modules/Bio/EnsEMBL/Analysis/Hive/Config/FinaliseCoreDB.pm

```diff
@@ -668,63 +710,410 @@ sub pipeline_analyses {
+  -flow_into => {
+    1 => ['select_busco_lineage']
+  },
+},
+
+
+{
+  -logic_name => 'select_busco_lineage',
+  -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveRunExternalCmd',
+  -parameters => {
+    indir    => $self->o('output_path'),
+    infile   => 'taxon.txt',
+    cmd      => 'python ' . $self->o('busco_lineage_selector_script') .
+                ' -d ' . $self->o('busco_datasets_file') .
+                ' -t ' .'$'.'(cat '.catfile($self->o('output_path'), 'taxon.txt'). ')',
+},
+  -rc_name => 'default',
+  -flow_into => {
+    1 => { 'run_busco_genome' => { lineage => '#stdout#' } },
+  },
+},
+
+     {
+       -logic_name => 'run_busco_genome',
+       -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+       -parameters => {
+	 cmd => 'cd '. $self->o('output_path') . ';' .
+	 'singularity exec ' . $self->o('busco_singularity_image') . ' busco -f '. '-i '. $self->o('output_path') . 'genome_dumps/' . $self->o('species_name').'_toplevel.fa '.'-m genome -l #lineage# -c 30 ' . '-o busco_genome ' . '--offline --download_path '. $self->o('busco_download_path') . ';' 
+      },
+           -rc_name => '120GB_30cpus',
+           -flow_into => {
+              1 => {'run_busco_protein' => {lineage => '#lineage#'} },
+      },
+   },
 
     {
-      -logic_name => 'dump_geneset_gtf',
-      -module     => 'Bio::EnsEMBL::Production::Pipeline::FileDump::Geneset_GTF',
-      -parameters => {
-        reg_conf         => $self->o('registry_file'),
-        species          => $self->o('production_name'),
-        custom_filenames => {
-          genes => $self->o('output_path') . '/' . $self->o('production_name') . '.gtf'
-        },
+       -logic_name => 'run_busco_protein',
+       -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+       -parameters => {
+         cmd => 'cd '. $self->o('output_path') . ';' .
+	 'singularity exec ' . $self->o('busco_singularity_image') . ' busco -f '. '-i '. $self->o('gst_dir') . '/' . $self->o('production_name') . '_protein_sequences.fa ' . '-m protein -l #lineage# -c 30 ' . '-o busco_protein '. '--offline --download_path '. $self->o('busco_download_path') . ';' 
       },
-      -rc_name   => '4GB',    # Adjust this based on your resource class
-      -flow_into => {
-        1 => ['dump_geneset_gff'], },
+           -rc_name => '80GB_30cpus',
+           -flow_into => {
+              1 => ['load_genome_busco_into_core'],
+      },
+   },
+
+{
+        -logic_name => 'load_genome_busco_into_core',
+          -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+          -parameters => {
+          cmd => 'python ' . $self->o('busco_metakeys_patch_script') .
+              ' -db '. $self->o('reference_db_name') . ' ' .  
+			  ' -host ' .  $self->o('reference_db_host') . ' ' .
+              ' -port ' . $self->o('reference_db_port') . ' ' .
+              ' -user ' . $self->o('user') . 
+              ' -password ' . $self->o('password') . 
```

Comment:
I think we have an issue broader than this PR with the passing of passwords via the CLI? I know it has always be done this way and the passwords arent exactly secrets but it would be better practice.